src/Uno.UWP/ApplicationModel/Email/EmailManager.iOS.cs

```diff
@@ -34,19 +34,17 @@ private static async Task ShowComposeNewEmailInternalAsync(EmailMessage message)
 			{
 				//fallback when user hasn't set up e-mail account yet
 				await ComposeEmailWithMailtoUriAsync(message);
-			}			
+			}
 		}
 
+#if !__MACCATALYST__
```

Comment:
We likely can include it for catalyst now, though if not fixed the exception message would have been required to be kept.

Reply:
@jeromelaban I just kept the existing behavior. This method is called only under `#if !__MACCATALYST__`, so the code under `#if __MACCATALYST__` can never execute.